src/routes/MapExplorer/MapExplorer.js

```diff
@@ -252,13 +273,66 @@ export default class MapExplorer extends React.Component {
     this.setState({ showFeatureFilters: !this.state.showFeatureFilters })
   }
 
+  toggleSavedSearches = (e) => {
+    this.setState({ showSavedSearches: !this.state.showSavedSearches })
+  }
+
+  saveSearch = (name) => {
+    const { currentBoundingBox } = this.state
+    const { currentUser, filters, location, posts } = this.props
+    const { featureTypes, search: searchText, topics } = filters
+    const { pathname } = location
+
+    let context
+    if (isAllCommunitiesPath(pathname)) context = 'all'
```

Comment:
yeah seems like context should be known and passed in from the .connector instead of determined everytime a search is saved here

Reply:
Yes - @lorenjohnson also commented on this above, I see that you'd mapped c to community and n to network as the subject, so I'm passing that in as the context.